seqr/views/apis/variant_search_api_2_3_tests.py

```diff
@@ -140,7 +143,8 @@ def test_query_variants(self, mock_get_variants, mock_get_gene_counts):
         export_url = reverse(export_variants_handler, args=[SEARCH_HASH])
         response = self.client.get(export_url)
         self.assertEqual(response.status_code, 200)
-        export_content = [row.split('\t') for row in response.content.rstrip('\n').split('\n')]
+        content = str(response.content, 'utf-8')
```

Comment:
its concerning to me that you have to re-encode the content to get it to look the way we want for the test - ideally in the test I would like to test what is actually being returned from the server. If the content of the response needs to change, I would rather change the api itself than the test. If the content of the response is good as is, I want to test what is actually being returned

Reply:
It makes sense. Changed to encode the expected data rather than to decode the result data.